src/Raven.Server/Documents/Replication/Incoming/IncomingReplicationHandler.cs

```diff
@@ -920,100 +316,29 @@ public LiveReplicationPerformanceCollector.ReplicationPerformanceType GetReplica
                 : LiveReplicationPerformanceCollector.ReplicationPerformanceType.IncomingExternal;
         }
 
-        public bool IsDisposed => _disposeOnce.Disposed;
+        protected override void OnAttachmentStreamsReceives(int attachmentStreamCount) => AttachmentStreamsReceived?.Invoke(this, attachmentStreamCount);
```

Comment:
we are calling those methods in Client API, InvokeXYZ, e.g. InvokeOnAttachmentStreamsReceived, maybe lets stick to that

Reply:
done